docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:xenial
+
+RUN apt-get update && apt-get install -y \
+        curl \
+        lsb-release \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl -sS https://dl.google.com/cloudagents/install-logging-agent.sh | bash
```

Comment:
Any reason why all of these runs can't be chained together into one long sequence? It would reduce the amount of layers for the docker image.